src/components/navMenu/NavMenu.tsx

```diff
@@ -33,6 +33,12 @@ export const NavMenu = ({
   const [selectedKey, setSelectedKey] = React.useState<string>(
     defaultSelectedKeys?.[0] || ''
   )
+  const [isToggleTooltipOpen, setToggleTooltipOpen] = React.useState(false)
+
+  const toggleCollapse = React.useCallback(() => {
+    setIsCollapsed(!isCollapsed)
+    setToggleTooltipOpen(false)
+  }, [isCollapsed])
```

Comment:
I hate that I have to manage the state of the Tooltip but it just was not liking something about how it's being used.